config/aws_config.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+type AWSConfig struct {
+	Ami                   string   `json:"ami,omitempty" yaml:"ami,omitempty"`
+	AWSAccessKey          string   `json:"awsAccessKey" yaml:"awsAccessKey"`
+	AWSInstanceType       string   `json:"awsInstanceType" yaml:"awsInstanceType"`
+	AWSRootSize           int64    `json:"awsRootSize" yaml:"awsRootSize"`
+	AWSSecretKey          string   `json:"awsSecretKey" yaml:"awsSecretKey"`
+	AWSSecurityGroupNames []string `json:"awsSecurityGroupNames" yaml:"awsSecurityGroupNames"`
+	AWSSecurityGroups     []string `json:"awsSecurityGroups" yaml:"awsSecurityGroups"`
+	AWSSubnetID           string   `json:"awsSubnetID" yaml:"awsSubnetID"`
+	AWSSubnets            []string `json:"awsSubnets" yaml:"awsSubnets"`
+	AWSVpcID              string   `json:"awsVpcID" yaml:"awsVpcID"`
+	AWSZoneLetter         string   `json:"awsZoneLetter" yaml:"awsZoneLetter"`
+	PrivateAccess         bool     `json:"privateAccess" yaml:"privateAccess"`
```

Comment:
I would think that some of these aren't required / could be null and therefore could have omitempty? Not entirely sure though and will leave this up to you

Reply:
Resolved. I honestly forgot to update this lol